quizcon/main/models.py

```diff
@@ -73,6 +140,22 @@ class QuestionResponse(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     modified_at = models.DateTimeField(auto_now=True)
 
+    def score_question(self):
+        correct_marker = self.correct_marker_position()
+        selected_position = self.selected_position
```

Comment:
This line seems a bit unnecessary. You can refer to `self.selected_position` throughout.